packages/scxa-tsne-plot/__test__/ClusterTSnePlot.test.js

```diff
@@ -49,6 +49,17 @@ describe(`ClusterTSnePlot colourize function`, () => {
     })
   })
 
+  test(`must move behind if metadata is Not Available`, () => {
+    const randomSeries = randomHighchartsSeriesWithNamesAndMaxPoints([...seriesNames, `Not available`], maxPointsPerSeries)
+    _colourizeClusters([], `lightgrey`)(randomSeries).forEach((series) => {
+      if(series.name === `Not available`) {
+        expect(series).toHaveProperty(`zIndex`)
```

Comment:
Since we want the “Not available” series to be behind the other series, the test should check that the `zIndex` value is negative.